src/Http/RouteConfig.php

```diff
@@ -8,18 +8,55 @@
 
 final class RouteConfig
 {
+    /** @var string The mark to give the next route in the matching Regex */
+    private string $regexMark = 'a';
+
     public function __construct(
         /** @var array<string, array<string, \Tempest\Http\Route>> */
-        public array $routes = [],
+        public array  $staticRoutes = [],
+        /** @var array<string, array<string, \Tempest\Http\Route>> */
+        public array  $dynamicRoutes = [],
+        /** @var array<string, string> */
+        public array $matchingRegexes = []
```

Comment:
Could we make this one private? Would that still work with the var exporter?

Regardless of whether that's possible, I'd move it outside of the constructor, as this is a variable we won't ever set from the outside — as far as I understand. I want to avoid confusion where possible.

Reply:
Making it private is not possible because is needed in GenericRouter. Should I add a getter and make the property private?

I have moved it outside of the constructor.

Reply:
It's fine for now. Let's just hope the property hooks RFC passes so that we can get a bit more control over public properties :)